src/carts/src/main/java/mushop/carts/CartService.java

```diff
@@ -0,0 +1,171 @@
+package mushop.carts;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Set;
+
+import io.helidon.common.http.Http;
+import io.helidon.common.http.Http.RequestMethod;
+import io.helidon.config.Config;
+import io.helidon.webserver.Routing.Rules;
+import io.helidon.webserver.ServerRequest;
+import io.helidon.webserver.ServerResponse;
+import io.helidon.webserver.Service;
+
+public class CartService implements Service {
+    
+    /** A reserved dbname to use for an in-memory repository instead of the autonomous database */
+    private static final String MOCKDB = "Mock";
```

Comment:
IIRC the value used is `"mock"` (case-sensitive?)

Reply:
should be case in-sensitive

Reply:
Fixed.